components/script/dom/bindings/structuredclone.rs

```diff
@@ -72,56 +72,36 @@ struct StructuredCloneWriter {
 }
 
 impl StructuredCloneWriter {
-    unsafe fn write_slice(&self, v: &[u8]) {
-        // writers bytesl takes in a vec! and its length
+    unsafe fn write_slice(&self, s: &usize, v: &[u8]) {
```

Comment:
We should not need to take an extra argument for the size. We should use `v.len()`.